README.md

```diff
@@ -62,28 +62,38 @@ For more information, see the [.NET Foundation Code of Conduct](https://dotnetfo
 ## Examples
 
 Here's an example of code to train a model to predict sentiment from text samples. 
-(You can see the complete sample [here](test/Microsoft.ML.Tests/Scenarios/SentimentPredictionTests.cs)):
+(You can find a sample of the legacy API [here](test/Microsoft.ML.Tests/Scenarios/SentimentPredictionTests.cs)):
 
 ```C#
-var pipeline = new LearningPipeline();
-pipeline.Add(new TextLoader(dataPath).CreateFrom<SentimentData>(separator: ','));
-pipeline.Add(new TextFeaturizer("Features", "SentimentText"));
-pipeline.Add(new FastTreeBinaryClassifier());
-var model = pipeline.Train<SentimentData, SentimentPrediction>();
+var env = new LocalEnvironment();
+var reader = TextLoader.CreateReader(env, ctx => (
+        Target: ctx.LoadFloat(2),
+        FeatureVector: ctx.LoadFloat(3, 6)),
+        separator: ',',
+        hasHeader: true);
+var data = reader.Read(new MultiFileSource(dataPath));
+var regression = new RegressionContext(env);
+var learningPipeline = reader.MakeNewEstimator()
+    .Append(r => (
+    r.SentimentText,
+    Prediction: regression.Trainers.Sdca(label: r.SentimentText, features: r.FeatureVector.Normalize())));
+var model = learningPipeline.Fit(data);
+
 ```
 
 Now from the model we can make inferences (predictions):
 
 ```C#
-SentimentData data = new SentimentData
+var predictionFunc = model.MakePredictionFunction<SentimentInput, SentimentPrediction>(env);
+var prediction = predictionFunc.Predict(new SentimentData
 {
     SentimentText = "Today is a great day!"
 };
-
-SentimentPrediction prediction = model.Predict(data);
-
 Console.WriteLine("prediction: " + prediction.Sentiment);
 ```
+For a complete CookBook of the new API you can check [here](docs/code/MlNetCookBook.md)
```

Comment:
Maybe "A cookbook that shows how to use these APIs for a variety of existing and new scenarios can be found [here](docs/code/MlNetCookBook.md)."